src/sema/assembly/mod.rs

```diff
@@ -6,12 +6,12 @@ use crate::sema::expression::ExprContext;
 use crate::sema::symtable::{LoopScopes, Symtable};
 use solang_parser::pt;
 
-mod block;
-mod builtin;
-mod expression;
+pub mod block;
+pub mod builtin;
+pub mod expression;
 mod for_loop;
-mod functions;
-mod statements;
+pub mod functions;
+pub mod statements;
```

Comment:
You're having to make these modules `pub` because the ast definition is scattered over sema now. The original ast had a definition in a single file, `ast.rs`.

Reply:
The AST for YUL is scattered across many files in `sema/assembly`. I can either create a new file `sema/assembly/ast.rs` to concentrate every AST structure or I can move them all to `sema/ast.rs`. Do you have any preference?